tests/testthat/test-with.R

```diff
@@ -14,20 +14,18 @@ test_that("linters_with_defaults warns on unused NULLs", {
 test_that("all default linters are tagged default", {
   expect_named(linters_with_defaults(), available_linters(tags = "default")$linter)
 
-  # TODO(michaelchirico): use plain expect_equal after waldo#133 makes it into a CRAN release
-  # Here, the environment()s are different because factories use them.
-  skip_if_not_r_version("4.1.0") # Desired all.equal behaviour only available in >= 4.1
+  # needed for {waldo} 0.4.0 (containing waldo#133)
```

Comment:
slightly puzzling to see a comment related to waldo followed by a skip on testthat version -- why not `skip_if_not_installed("waldo", "0.4.0")`?